packages/preview/cqut-thesis-typst/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name        = "cqut-thesis-typst"
```

Comment:
Unless this template is official and endorsed by the university staff, you should change the name of the package to something more unique (you can add a distinctive adjective for instance). If someone else was to submit another non-official alternative to this package, they should not be disadvantaged because you already took the "canonical" name.
If this template is an official one, please tell me by writing an email to `hello@typst.app` stating so, using your academic email.

And regardless of whether the package is official or not, its name should not include `typst`: it is obvious that this is a Typst package, so the information is just redundant.

Can you please rename it to conform to our guidelines?

Reply:
Thanks for your guide.

I've renamed the package name from "cqut-thesis-typst" to "modern-cqut-thesis" , and used the lastest package...

![image](https://github.com/user-attachments/assets/baa069f8-745b-47ce-92d9-f243adc8f5cd)

If there any problem feel free to point out.